cardano-testnet/test/Test/FoldBlocks.hs

```diff
@@ -35,13 +35,12 @@ instance Show FoldBlocksException where
 -- that main thread blocks on.
 prop_foldBlocks :: H.Property
 prop_foldBlocks = H.integrationRetryWorkspace 2 "foldblocks" $ \tempAbsBasePath' -> do
-
   -- Start testnet
   base <- HE.noteM $ liftIO . IO.canonicalizePath =<< HE.getProjectBase
-  configTemplate <- H.noteShow $ base </> "configuration/defaults/byron-mainnet/configuration.yaml"
+  configTemplate <- H.noteShow $ base </> "configuration/defaults/byron-mainnet/configuration.yaml" --- outside of the package directory !!
   conf <- HE.noteShowM $
     TN.mkConf (TN.ProjectBase base) (TN.YamlFilePath configTemplate)
-      (tempAbsBasePath' <> "/")
+      (tempAbsBasePath' <> "/") --  '<> "/")'  is needed otherwise test fails !
```

Comment:
I know this is old code, but can you find out why?

Reply:
I only stumbled upon this because `(tempAbsBasePath' <> "/")` looks strange.
but as the comment says the tests fail without the trailing `<> "/"`.
I'm in the course of reordering the various file paths and this will probably be removed anyway.